src/emqttd_gc.erl

```diff
@@ -0,0 +1,47 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2013-2017 EMQ Enterprise, Inc. (http://emqtt.io)
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+%% GC Utility functions.
+
+-module(emqttd_gc).
+
+-author("Feng Lee <feng@emqtt.io>").
+
+-export([conn_max_gc_count/0, reset_conn_gc_count/2, maybe_force_gc/2]).
+
+-spec(conn_max_gc_count() -> integer()).
+conn_max_gc_count() ->
+    case emqttd:env(conn_force_gc_count) of
+        {ok, I} when I > 0 -> I + rand:uniform(I);
```

Comment:
Maybe this random value varies too much, and it is hard to compare performances of this parameter.